apps/game_client/lib/game_client_web/router.ex

```diff
@@ -24,8 +24,8 @@ defmodule GameClientWeb.Router do
     pipe_through :browser
     # pipe_through [:browser, :game]
 
-    live "/board/:game_id/:player_id", BoardLive.Show
-    live "/board/:player_id", BoardLive.GameQueue
+    live "/board/play/:game_id/:player_id", BoardLive.Show
+    live "/board/:player_id/:character", BoardLive.GameQueue
```

Comment:
Maybe we should update the readme file since it includes this route and it's no longer working

Reply:
fixed 👍